### ISSUE TITLE
SE-MODEL: state estimation, calibration, model discrepancy, and multi-fidelity model evolution

### ISSUE THREAD

**Issue by Tristan-Stoltz-ERC** [Luminous-Dynamics/symthaea#3759]
## Goal

Add an evidence-bounded model-update discipline connecting SE-OBS, SE-OPT, SE-VV, SE-SYNTH and digital twins without allowing telemetry, calibration, data assimilation, surrogate fitting or multi-fidelity fusion to silently rewrite a qualified engineering model.

Parent programs: #3677, #3686, #3695, #3697, #3701.

## Core distinctions

```text
measurement != inferred state
inferred state != calibrated parameter
calibrated parameter != validated parameter
parameter posterior != model-form correction
good calibration fit != predictive validation
surrogate prediction != high-fidelity result
multi-fidelity fusion != consensus truth
data assimilation update != evidence admission
persistent residual != automatically identified cause
```

### 1. State estimation

Consumes observations to estimate the current latent/as-operated system state while keeping the underlying engineering model revision fixed.

Examples:
- Kalman / ensemble / particle-style filters;
- state observers;
- smoothing / reanalysis;
- digital-twin state estimation.

Output is a versioned `StateEstimate`, not a physical observation.

### 2. Parameter calibration

Uses a declared calibration dataset and exact model revision to estimate uncertain model parameters.

Calibration must preserve:
- calibration dataset identity;
- priors/bounds;
- likelihood/error model;
- optimizer/sampler identity;
- seeds and stopping rules;
- posterior/point-estimate identity;
- identifiability diagnostics;
- correlations;
- residuals;
- model-discrepancy assumptions.

A calibrated parameter set is a candidate model revision, not a validated truth.

### 3. Model-form revision

Persistent, structured discrepancy that cannot be explained by state uncertainty, parameter uncertainty, numerical error, measurement error or known validity limits may generate a `ModelFormReviewRequest`.

No residual-analysis component may silently add equations, latent causes, force terms, constraints or couplings to the qualified model.

## Proposed types

```text
StateEstimate
StateEstimateRevision

CalibrationDataset
CalibrationCampaign
CalibrationResult
ParameterPosterior
ParameterRevisionProposal

ModelDiscrepancy
DiscrepancyPattern
ModelFormReviewRequest

FidelityLevel
ModelFidelityDescriptor
MultiFidelityDataset
SurrogateModelRevision
SurrogateApplicabilityEnvelope

AssimilationCampaign
AssimilationUpdate

ValidationHoldoutPolicy
ModelUpdateAssessment
```

## Calibration/validation split

Require explicit non-overlap where feasible:

```text
calibration data
    -> parameter proposal

held-out validation data
    -> predictive assessment

operational data
    -> current-state estimation / later applicability review
```

If the same data must serve multiple purposes, record the dependence and lower the admissible claim ceiling rather than pretending independence.

Core theorem:

```text
posterior fit on calibration data != predictive validation
```

## Model discrepancy budget

Retain uncertainty/error classes separately:

- observation/measurement uncertainty;
- input uncertainty;
- numerical/discretization uncertainty;
- parameter uncertainty;
- initial/state uncertainty;
- stochastic variability;
- surrogate error;
- model-form discrepancy;
- unknown/unmodeled remainder.

Do not collapse these into a universal confidence scalar.

## Multi-fidelity model contract

Every fidelity level must have exact model/tool/configuration identity.

```text
low-fidelity model
high-fidelity model
physical experiment
surrogate
```

are distinct sources, not interchangeable truth levels.

A multi-fidelity surrogate must retain:

- all training-source identities;
- fidelity ordering/rationale;
- training points and units;
- hyperparameters;
- correlation/discrepancy structure;
- cross-validation diagnostics;
- prediction uncertainty;
- interpolation/extrapolation status;
- applicability envelope;
- stale-input/model detection.

```text
cheap model correlated with expensive model
!= cheap model physically valid
```

and

```text
surrogate uncertainty small
!= model-form uncertainty small
```

### Dakota

Primary calibration/UQ campaign engine where appropriate. Dakota supports parameter estimation through deterministic nonlinear least squares or Bayesian inference and broader UQ/sensitivity workflows.

### OpenMDAO

Use existing multi-fidelity surrogate support where it fits multidisciplinary analysis. Preserve the exact high- and low-fidelity datasets rather than treating the surrogate as the canonical model.

### SMT

Optional specialist surrogate/multi-fidelity adapter after the common contract qualifies. Useful for kriging, variable-fidelity modeling and Bayesian/efficient-global-optimization research.

### DART

Optional later data-assimilation adapter for ensemble DA / observing-system experiments where the domain and scale justify it. Do not make a geoscience-oriented package the mandatory core state-estimation implementation.

### SE-MODEL-000 — model-update architecture profile
Docs only. Freeze the state/calibration/model-form split and authority boundaries.

### SE-MODEL-001 — state-estimate contract
Typed inferred-state revisions bound to exact observations, model revision, configuration and algorithm identity.

### SE-MODEL-002 — calibration campaign/result contract
Exact dataset/model/prior/likelihood/algorithm lineage. No model mutation.

### SE-MODEL-003 — calibration-to-revision proposal
Turn a qualified calibration result into a `ParameterRevisionProposal`; separate review/acceptance required before a new model revision exists.

### SE-MODEL-004 — discrepancy ledger
Classify residuals by candidate source without auto-selecting a cause.

### SE-MODEL-005 — held-out validation boundary
Bind calibration and validation datasets and detect leakage / reuse / dependence.

### SE-MODEL-006 — multi-fidelity source + surrogate contract
Typed fidelity identities, training lineage, uncertainty and applicability envelope.

### SE-MODEL-007 — Dakota adapter
Bayesian/deterministic calibration and UQ campaigns through SE-FOSS execution provenance.

### SE-MODEL-008 — OpenMDAO / SMT multi-fidelity adapters
Surrogate fitting/prediction with exact source-data lineage and extrapolation flags.

### SE-MODEL-009 — observation assimilation
State-estimation / filtering over SE-OBS observations while preserving observation != estimate.

### SE-MODEL-010 — model-form review proposal
Persistent unexplained discrepancy may request model revision; HDC/LTC/causal reasoning remain proposal-only.

### SE-MODEL-011 — optimal experiment design
Use uncertainty/discrepancy to propose the next informative experiment, but keep expected information gain / search utility separate from evidence.

## Qualification corpus

Include:
- perfectly identifiable calibration;
- deliberately non-identifiable parameters;
- calibration overfit that fails held-out validation;
- biased measurements;
- stale calibration;
- correlated errors;
- wrong model form with parameter compensation;
- surrogate interpolation vs extrapolation;
- low/high-fidelity disagreement;
- changing operational state mistaken for parameter drift;
- parameter drift mistaken for model-form error;
- synthetic data presented as physical calibration data;
- duplicate observations counted as independent;
- validation-data leakage into calibration/training.

## Ordering

Do not implement on the unqualified SE-001 semantic surface.

Recommended dependency:

```text
SE-001 qualification
-> SE-002A descriptive composition
-> SE-OBS ObservationEnvelope
-> SE-FOSS EngineeringArtifact / ExecutionCapsule
-> SE-VV credibility primitives
-> SE-MODEL-001/002
-> calibration / assimilation / multifidelity adapters
-> active experiment design
```

## Non-goals

- no automatic self-calibration of accepted engineering models;
- no telemetry -> model rewrite shortcut;
- no posterior -> physical truth shortcut;
- no surrogate -> high-fidelity evidence substitution;
- no residual -> causal diagnosis shortcut;
- no optimization/search utility -> evidence strength shortcut;
- no universal model-confidence scalar;
- no ETK replacement.

## Exit criterion

Symthaea can distinguish **what was measured, what system state is inferred, which parameters were calibrated, where the model still disagrees with reality, how multiple fidelity levels relate, and when a model revision should merely be proposed**, while preserving independent validation and ETK authority boundaries.

**Comment by Tristan-Stoltz-ERC** [comment]
Deeper source audit / refinement before implementation:

## Existing digital-twin migration boundary

Current `symthaea-digital-twin::TwinState::ingest_with_uncertainty` is useful exploratory machinery, but it is **not** the future SE-MODEL assimilation/calibration authority path.

Concrete current behavior on `main`:

- `TelemetryPoint` carries string channel/unit + UTC timestamp rather than SE-OBS provenance/configuration/calibration identity;
- `expected_value` and `tolerance` are passed ad hoc rather than bound to an exact model/configuration revision;
- epistemic/aleatoric values are exponentially smoothed heuristics;
- `health` is derived directly from the free-energy proxy;
- `needs_intervention` is a generic scalar threshold;
- when `tolerance.abs() <= f64::EPSILON`, normalized error becomes **0.0**, which is acceptable defensive prototype behavior but must fail/diagnose explicitly in engineering evidence mode;
- `TelemetryPoint::now` uses wall-clock time and must not be the canonical observation identity path.

Therefore preserve this crate as a lightweight exploratory/proposal layer while SE-OBS/SE-MODEL introduce exact observation/state-estimation contracts.

Required theorem:

```text
TwinState heuristic update != state-estimation evidence
health proxy != validated asset health
free-energy threshold != safety limit
```

## Existing Futures Laboratory calibration is a different concept

`symthaea-futures-calibration` implements proper scoring / forecast-calibration diagnostics (Brier, CRPS, log-score, reliability curves). That is valuable for **probabilistic prediction calibration**, but it must not be confused with physical-model parameter calibration / inverse problems.

Potential reuse:
- score probabilistic engineering forecasts;
- reliability diagrams / calibration diagnostics;
- evaluate predictive distributions from model ensembles.

Do not reuse the name/API as the parameter-calibration contract.

```text
forecast probability calibration
!= parameter estimation
!= physical model validation
```

## Add a shadow-model promotion path

Operational adaptation should never mutate the current accepted model in place.

Preferred lifecycle:

```text
observations / calibration campaign
-> ParameterRevisionProposal or ModelFormReviewRequest
-> ShadowModelRevision
-> frozen historical replay
-> independent held-out validation
-> applicability / change-impact review
-> ETK currentness consequences
-> explicit human/institutional promotion
-> NewModelRevision

old revision remains addressable for rollback/audit
```

Candidate additional types:

```text
ShadowModelRevision
ApplicabilityEnvelope
ApplicabilityAssessment
DomainShiftReceipt
ModelPromotionReceipt
RollbackReference
```

## Out-of-domain behavior

Before auto-calibration, ask whether the current asset/environment has left the model's validated envelope.

```text
large residual
-> check observation quality
-> check numerical health
-> check state uncertainty
-> check parameter uncertainty
-> check applicability / domain shift
-> only then calibration or model-form review
```

Out-of-envelope operation should normally produce `ApplicabilityReviewRequired`, not automatic parameter fitting.

## Holdout discipline

For temporal/operational systems, validation partitioning must be time-aware where appropriate. Future observations must never leak into calibration or surrogate training for an earlier prediction claim.

Track at least:
- calibration/training interval;
- validation interval;
- operational evaluation interval;
- overlap/dependence;
- reuse history.

This should become a first-class anti-leakage fixture in SE-MODEL-005.

**Comment by Tristan-Stoltz-ERC** [comment]
Additional reuse audit: `symthaea-futures-state` already reuses `symthaea-fep::ActiveInferenceAgent` / `HiddenState` as a real belief-state estimator and has a useful leakage-safe observation masking primitive.

However its own source documents important gaps for engineering assimilation:
- observations are dense fixed-width vectors with no native missing-data mask;
- observation timestamps are not consumed by `perceive()`; internal time advances per call;
- per-observation `precision` is currently not used by the FEP computation;
- the masking layer is designed for controlled Futures-Lab visibility, not calibrated sensor error models.

Therefore:

```text
FEP belief update
!= universal engineering state estimator
```

but it can be one qualified/experimental estimator behind a common contract.

Suggested common interface concepts:

```text
StateEstimatorDescriptor {
    estimator_id
    algorithm_family
    supports_missing_data
    supports_irregular_time
    supports_nonlinear
    supports_non_gaussian
    state_dimension_policy
    observation_model_requirements
    uncertainty_representation
    known_assumptions[]
}

StateEstimationReceipt {
    prior_state_ref
    model_revision
    configuration_revision
    observation_set_refs[]
    process_model_ref
    observation_model_ref
    estimator_descriptor
    time_window
    posterior_state
    posterior_uncertainty
    innovation/residual diagnostics
    filter_health
    warnings[]
}
```

Potential implementations can include:
- conventional Kalman / EKF / UKF where assumptions fit;
- ensemble / particle methods;
- FEP / active-inference estimator;
- domain-specific observers;
- optional later DART adapter for large ensemble-DA problems.

If an algorithm performs **joint state + parameter estimation**, split the outputs:

```text
posterior state
-> StateEstimateRevision

posterior parameter proposal
-> ParameterRevisionProposal
```

Never mutate accepted model parameters merely because the estimator supports dual estimation.

Also keep Futures forecast calibration reusable only for scoring probabilistic outputs; its `BeliefState` / forecast scoring lineage remains distinct from physical-model calibration and SE-VV validation.

**Comment by Tristan-Stoltz-ERC** [comment]
Cross-link: #3779 separates **model applicability** from **physical operating limits** and **recoverability**.

A large residual should not jump directly to fault/reversion semantics without checking observation quality, state uncertainty and model/domain applicability.

```text
out of model envelope != automatically physically unsafe
```

but loss of model credibility may justify a lower runtime authority ceiling under an independently defined RTA policy.
